CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 1.0.0-beta.65
+
+### Features
+
+Remove strikethru line from disabled `MegaButton`
+Add image opacity to disabled `MegaButton`
```

Comment:
wording nits: 'strikethrough' may look better? 
and I would say we 'reduce' the opacity of a disabled megabutton (not add)